src/util/VanDerCorput.cpp

```diff
@@ -116,10 +131,8 @@ VanDerCorput::const_iterator VanDerCorput::end() const
 }
 
 //=============================================================================
-VanDerCorput::const_iterator::const_iterator(const VanDerCorput *seq)
-    : mSeq(seq)
-    , mN(-1)
-    , mFinalIter(false)
+VanDerCorput::const_iterator::const_iterator(const VanDerCorput* seq)
+  : mSeq(seq), mN(-1), mFinalIter(false)
```

Comment:
ClangFormat (at least version 3.8) doesn't support enforcing initialization list to take each line per item. The only related option is `ConstructorInitializerAllOnOneLineOrOnePerLine`, but it just enforces it only when it doesn't fit on a line.

Regarding where to break line in initialization line, there is an option `BreakConstructorInitializersBeforeComma` exactly what you want. I didn't turn it on though. :smiling_imp: I could get used to breaking-constructor-initializers-before-comma style if I have to, but I'd like to vote on not to do.